torch/_inductor/custom_partitioner_fn.py

```diff
@@ -0,0 +1,54 @@
+from abc import ABC, abstractmethod
+from collections.abc import Sequence
+from typing import Any, Optional
+from typing_extensions import TypeAlias
+
+import torch
+
+
+class CustomPartitionerFn(ABC):
```

Comment:
nit: I would probably bundle this into the same file that the other `CustomGraphPass` object definitions live (`custom_graph_pass.py`), since at the end of the day the purpose of the extension is the same (custom graph pass accepting (graph, args)), just with a slightly different signature